translations/client_de.ts

```diff
@@ -6291,4 +6291,4 @@ Server antwortete mit folgendem Fehler: %2</translation>
         <translation>Ich habe etwas mit Ihnen geteilt</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
```

Comment:
@tobiasKaminsky this must have a new line at the end, with that said, do you even need to change this file?

Reply:
No.
I can revert it. Actually I have no idea where this comes from…

Reply:
Fixed it